### DISCUSSION TITLE
Asset check from CLI

### DISCUSSION THREAD

**Discussion by dridk** [dagster-io/dagster#17566] [Ideas]
I don't use dagit when I code an asset. I always run asset materialisation from the command line. It would be greate to also run check from the cli. For instance, the following command will materialize and check the asset 

```
dagster asset check -n myasset 

```

Or maybe, just add a flag within materialize command 

```
dagster asset materialize -n myasset --check
```

**Comment by sryza** [comment]
Hey @dridk - this seems sensible. I'm going to convert it to a Github Issue for tracking.

**Comment by sryza** [comment]
Issue link: https://github.com/dagster-io/dagster/issues/17606
